substrate/frame/election-provider-multi-phase/test-staking-e2e/src/mock.rs

```diff
@@ -77,7 +78,7 @@ frame_support::construct_runtime!(
 	}
 );
 
-pub(crate) type AccountId = u64;
+pub(crate) type AccountId = u128;
```

Comment:
for some reason this fails to compile complaining account id expected as u64 at someplace. I think one of the impls used in the runtime must have u64 hardcoded but the rust err is not very helpful.

Reply:
The `frame_system` pallet;s config is using the default `config_preludes::TestDefaultConfig` impl which sets the `AccountId` type to u64. We need to overwrite it in the `frame_system::Config` in this tests to `u128`